### PR TITLE
fs-1819 Added import script.

### PR DESCRIPTION
**Pull request by SentientHousePlant**
This needs the application store PR to be merged first. 
- Adds the import script.
- Adjusts the current insert to do upserting during conflicts.

### CODE REVIEW

**Comment by gidsg** [comment]
Can we document how this process is invoked?

File: scripts/import_from_application.py [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
 with app.app_context():
 
-    # Grab data from app store.
+    applications_url = (
+        Config.APPLICATIONS_ENDPOINT
+        + "?status_only=SUBMITTED"
+        + f"?round_id={args.roundid}"
+    )
+
+    app_store_response_json = requests.get(applications_url).json
```
Comment (tferns) [inline scripts/import_from_application.py:24]:
```suggestion
    app_store_response_json = requests.get(applications_url).json()
```

**Review by tferns — CHANGES_REQUESTED** [review]
Changes look good but think there's a small error in your script

**Review by tferns — APPROVED** [review]
Looks good! Loving the small maintenance changes that are gradually making the code better

**Review by tferns — CHANGES_REQUESTED** [review]
Sorry to revoke my approval, I've noticed some issues when trying to run this script locally

I'll outline them in comments

**Review by tferns — CHANGES_REQUESTED** [review]
Forgot to submit these comments, my bad...

**Pull request merged**